cmd/repo.go

```diff
@@ -279,18 +282,23 @@ func getArtifactsNotIndexed(index repo.Index, distributedFiles string) repo.Inde
 	filesNotIndexed := getFilesNotIndexed(index, distributedFiles)
 
 	artifactsNotIndexed := make(repo.Index, 0)
+
 	for _, file := range filesNotIndexed {
 		artifactInfo := new(repo.ArtifactInfo)
 		artifactInfo.Filename = filepath.Base(file.Name())
 
 		if err := artifactInfo.SetInfoFromFileName(artifactInfo.Filename); err != nil {
-			continue
+			artifactInfo.Name = artifactInfo.Filename
+			artifactInfo.Version = "unknown"
+			artifactInfo.Arch = "unknown"
+			artifactInfo.OperatingSystemFamily = "unknown"
+			artifactInfo.OperatingSystem = "unknown"
```

Comment:
Why do you not want to skip artifacts when err != nil?

Reply:
> I don't want to rely on artifacts regexes...

If you don't want to rely on artifacts regexes, I think we need to accept all files in `/all/` even if they don't comply to one regex.
If you have another solution, please, explain.

Reply:
Accept all files /all/ that comply with regex?

I think it's a better idea to WARN if a foreign file/directory is found.

Reply:
Ok: let's accept the file but I'm going to add a WARN log!